lib/buffered_logger.rb

```diff
@@ -45,7 +41,17 @@ def started?
   end
 
   def sweep
-    @logdev.sweep
+    warn "[DEPRECATION] `sweep` is deprecated.  It is no longer necessary."
```

Comment:
Totally agree. Scrub it, especially because it doesn't even do what you want it to do anymore.